internal/services/cosmos/cosmosdb_account_resource.go

```diff
@@ -309,26 +354,25 @@ func resourceCosmosDbAccount() *pluginsdk.Resource {
 				Type:     pluginsdk.TypeSet,
 				Optional: true,
 				Computed: true,
-				ForceNew: true,
 				Elem: &pluginsdk.Resource{
 					Schema: map[string]*pluginsdk.Schema{
 						"name": {
 							Type:             pluginsdk.TypeString,
 							Required:         true,
 							DiffSuppressFunc: suppress.CaseDifference,
 							ValidateFunc: validation.StringInSlice([]string{
```

Comment:
@sivethe As mentioned in the email by Nanxuan as following, could you explain what the type of this request here?


1.	If for now only 1~2 enum would be added, I suggest to follow the current way that service team has TF team be aware of what those new values are, and again TF team needs to test those values before onboarding.
2.	If service team can predict such new capacities would occur frequently, it could be an open discussion to remove all validations on this specific case but service team need to guarantee the above 2 TF design principles.
	For example, as the current TF doc says “Only A,B,C can be added to an existing Cosmos DB Account” / “Only D can be removed from an existing Cosmos DB Account”. If there were the current fact, if there could be official and keep-updated document clarifying that or service api could fast fail users’ invalid input with returning friendly effort message, TF could further see what can move on.

Reply:
We validate as much as we can on the terraform side for a few reasons: it shows up in plan instead of during apply (which can happen hours into a tf apply), it ensure if a value is removed from an SDK w are aware and can check to see if it was supposed to be, and then update our docs etc. and does provide a better use experience for existing values. When new ones are added it is a small PR to update our validation/list which typically is merged and released within weeks.